### PR TITLE
Pan os -  test security policy

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19496

## Description
see design in issue
UC: test if a modification to a security policy is needed

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests - unit and tpb
- [ ] Documentation - soon
- [x] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [x] [YAML file](https://github.com/demisto/content/blob/dda6a54ab15c07b75eb1e59fdeec6d854d7f2b52/Integrations/Panorama/Panorama.yml)
- [x] [CHANGELOG](https://github.com/demisto/content/blob/4e8ce87bf449d90fd353e5e2c45291cbc3395ef8/Integrations/Panorama/CHANGELOG.md)

### CODE REVIEW

**Comment by kirbles19** [comment]
@bakatzir Done.

File: Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 category: Network Security
 commonfields:
-  id: Panorama
+  id: Panorama2
```
Comment (yuvalbenshalom) [inline Integrations/Panorama/Panorama.yml]: Panorama2 ?

File: Integrations/Panorama/Panorama.yml [hunk 2]
```diff
@@ -34,8 +34,8 @@ configuration:
   type: 0
 description: Manage Palo Alto Networks Firewall and Panorama. For more information
   see Panorama documentation.
-display: Palo Alto Networks PAN-OS
-name: Panorama
+display: Palo Alto Networks PAN-OS2
```
Comment (yuvalbenshalom) [inline Integrations/Panorama/Panorama.yml]: remove 2

File: Integrations/Panorama/Panorama.yml [hunk 3]
```diff
@@ -34,8 +34,8 @@ configuration:
...
+name: Panorama2
```
Comment (yuvalbenshalom) [inline Integrations/Panorama/Panorama.yml]: remove 2

**Pull request merged**